protocol/0042-LIQF-setting_fees_and_rewarding_lps.md

```diff
@@ -380,12 +380,15 @@ There is an example [google sheet for this step](https://docs.google.com/spreads
 
   - If an LP has an active liquidity provision at the start of an epoch and throughout the epoch always meets their liquidity provision requirements then they will have a `fraction_of_time_on_book == 1` and no penalty will be applied to their liquidity fee payments at the end of the epoch (<a name="0042-LIQF-035" href="#0042-LIQF-035">0042-LIQF-035</a>)
   - If an LP has an active liquidity provision at the start of an epoch and throughout the epoch meets their liquidity provision requirements less than `market.liquidity.commitmentMinTimeFraction` fraction of the time then they will have a full penalty and will receive `0` liquidity fee payments at the end of the epoch (<a name="0042-LIQF-049" href="#0042-LIQF-049">0042-LIQF-049</a>)
+  - If an LP has an active liquidity provision at the start of an epoch, during the epoch market parameter change `market.liquidity.performanceHysteresisEpochs > 1` is enacted, throughout the epoch always meets their liquidity provision requirements then they will have a `fraction_of_time_on_book == 1` and no penalty will be applied to their liquidity fee payments at the end of the epoch. The next epoch, the average `penalty rate` over the previous `n-1` epochs is `0.75` and throughout the epoch they always meet their liquidity provision requirements then they will have a `fraction_of_time_on_book == 1` for the latest epoch a penalty rate of `0.75` will be applied to liquidity fee payments at the end of the epoch. (<a name="0042-LIQF-054" href="#0042-LIQF-054">0042-LIQF-054</a>)
```

Comment:
Which part of the spec does ` then they will have a `fraction_of_time_on_book == 1` for the latest epoch a penalty rate of `0.75` will be applied to liquidity fee payments at the end of the epoch.` come from?

Reply:
that should be an assumption like so

 `If an LP has an active liquidity provision at the start of an epoch, the average `penalty rate` over the previous `n-1` epochs is `0.75` and during the epoch market parameter change `market.liquidity.performanceHysteresisEpochs > 1` is enacted, throughout the epoch always meets their liquidity provision requirements then they will have a `fraction_of_time_on_book == 1` and no penalty will be applied to their liquidity fee payments at the end of the epoch. The next epoch, the average `penalty rate` over the previous `n-1` epochs is `0.75` and throughout the epoch they always meet their liquidity provision requirements then they will have a `fraction_of_time_on_book == 1` for the latest epoch a penalty rate of `0.75` will be applied to liquidity fee payments at the end of the epoch. (<a name="0042-LIQF-054" href="#0042-LIQF-054">0042-LIQF-054</a>)`

Reply:
It's just not correct the way it's written.

Reply:
The relevant calculation is lines 264-275 in 0042.

Reply:
i spoke to David offline and merged his suggestion for this AC